core/server/master/src/test/java/alluxio/master/journal/ufs/UfsJournalLogWriterTest.java

```diff
@@ -283,6 +282,7 @@ public void missingJournalEntries() throws Exception {
       writer.write(newEntry(nextSN));
       nextSN++;
       if (i == 3) {
+        flushOutputStream(writer);
```

Comment:
Can you call `writer.flush` here instead?

Reply:
Done